<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductsRedactor</class>
 <widget class="QDialog" name="ProductsRedactor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="itemsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>название/цена/количество/кто добавил</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="addInput"/>
    </item>
    <item>
     <widget class="QLineEdit" name="deleteInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>81</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Убрать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>141</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="deleteAllButton">
      <property name="text">
       <string>Удалить со склада</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addNewTypeButton">
      <property name="text">
       <string>Добавить новый продукт</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Редактор товаров</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ProductsRedactor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
